DevCycle.SDK.Server.Cloud.MSTests/DevCycleTest.cs

```diff
@@ -50,7 +50,7 @@ public async Task GetProductionAllFeatures()
                     .SetSDKKey(Environment.GetEnvironmentVariable("DEVCYCLE_SERVER_SDK_KEY"))
                     .SetLogger(new NullLoggerFactory())
                     .Build();
-            var resp = await api.AllFeaturesAsync(new DevCycleUser("test"));
+            var resp = await api.AllFeatures(new DevCycleUser("test"));
```

Comment:
Are these API changes going to make this a major version update?

Reply:
Yes

Reply:
We also changed the Local signatures to return tasks to support async/await.  So that alone would require a major bump

Reply:
It's not recommended to do that in C# - you shouldn't have deprecated features if it's just a signature change. Functionality changes is where you'd add a deprecated function

Reply:
If you are not doing a major bump - yes but this is a major bump.